eth_utils/address.py

```diff
@@ -35,11 +35,12 @@ def is_hex_address(value):
         return False
 
 
-@coerce_args_to_bytes
-def is_binary_address(value):
+def is_binary_address(primitive=None, hexstr=None, text=None):
```

Comment:
Same here.  Function should take a single `value` parameter, return `False` if the value is not a bytes type, remove the `coerce_xxx` decorator.